### PR TITLE
Added new scream to the ushanka

### PR DESCRIPTION
**Pull request by Jujumatic**
Added a 20% chance to the ushanka giving a new cyka scream.

Mostly adding it as a joke and as late april fools

### CODE REVIEW

**Comment by JohnGinnane** [comment]
That certainly is a way to do what you want although i might suggest the following:

Typically we write `define`s at the top of the file, and in this case the defines are only needed in this file so at the bottom you can do `#undef <nameofdefine>`

I would also just do two defines possibly;

```
#define NORMAL_SCREAMS list(...)
#define SPECIAL_SCREAMS list(...)

// then further down where the ushanka is defined
/obj/item/clothing/head/ushanka
    alternate_screams = list(NORMAL_SCREAMS, NORMAL_SCREAMS, NORMAL_SCREAMS, SPECIAL_SCREAMS)
```

**Comment by Pyko1** [comment]
Not really big fan of this, due to it having actual music in it. -1

**Comment by Jujumatic** [comment]
I tried it without the music part and imo it really doesn't work nearly as well without the music

File: hippiestation/code/modules/clothing/head/misc.dm [hunk 1]
```diff
@@ -1,11 +1,14 @@
+#define normalscreams list('hippiestation/sound/misc/cyka1.ogg', 'hippiestation/sound/misc/cheekibreeki.ogg')
```
Comment (JohnGinnane) [inline hippiestation/code/modules/clothing/head/misc.dm:1]: why no caps? :(

**Comment by Jujumatic** [comment]
uh oh LOL i did it again, hey guys want 5 million commits merged?

**Comment by Jujumatic** [comment]
hhhhollyy fuck im gonna close this and fix this asap lmfao

**Pull request closed without merging**